packages/mobx-react/__tests__/observer.test.tsx

```diff
@@ -908,3 +908,71 @@ test("Missing render should throw", () => {
     }
     expect(() => observer(Component)).toThrow()
 })
+
+test("this.context is not observable if ComponentName.contextType is not set", () => {
+    const store = observable({ counter: 0 })
```

Comment:
You don't need any stores or observables here. All we have to test is:
Parent component has a state (eg number).
Update parent component state (eg increment the number).
Parent component passes this state (number) via context to child.
Child returns `this.context` from `@computed`.
Child returns `@computed` in render.
Prior fix the child's output won't correlate with parent's state.

Reply:
It's better now?